xapi-tracing.opam

```diff
@@ -14,6 +14,7 @@ depends: [
   "dune"
   "re"
   "uri"
+  "uuid" {with-test}
```

Comment:
If this is needed as a dependency for tests, why isn't alcotest needed as well?

Reply:
It probably is, you just happen to get it installed because another package requires it.

This is what `opam-dune-lint` says:
```
xapi-tracing-export.opam: changes needed:
  "ppx_deriving_rpc" {>= "9.0.0"}          [from ocaml/libs/tracing]
  "ptime" {>= "1.1.0"}                     [from ocaml/libs/tracing]
  "result" {>= "1.5"}                      [from ocaml/libs/tracing]
  "rresult" {>= "0.7.0"}                   [from ocaml/libs/tracing]
  "uri" {>= "4.4.0"}                       [from ocaml/libs/tracing]
  "alcotest" {with-test & >= "1.7.0"}      [from ocaml/libs/xapi-stdext/lib/xapi-stdext-threads, ocaml/tests/record_util]
  "astring" {with-test & >= "0.8.5"}       [from ocaml/tests/record_util]
  "fmt" {with-test & >= "0.9.0"}           [from ocaml/libs/xapi-stdext/lib/xapi-stdext-threads, ocaml/libs/xapi-stdext/lib/xapi-stdext-unix/test, ocaml/tests/record_util]
  "mtime" {with-test & >= "2.0.0"}         [from ocaml/libs/xapi-stdext/lib/xapi-stdext-threads, ocaml/libs/xapi-stdext/lib/xapi-stdext-unix/test]
  "qcheck-core" {with-test & >= "0.21.2"}  [from ocaml/libs/xapi-stdext/lib/xapi-stdext-unix/test]
xapi-tracing.opam: changes needed:
  "alcotest" {with-test & >= "1.7.0"}      [from ocaml/libs/xapi-stdext/lib/xapi-stdext-threads, ocaml/tests/record_util]
  "astring" {with-test & >= "0.8.5"}       [from ocaml/tests/record_util]
  "fmt" {with-test & >= "0.9.0"}           [from ocaml/libs/xapi-stdext/lib/xapi-stdext-threads, ocaml/libs/xapi-stdext/lib/xapi-stdext-unix/test, ocaml/tests/record_util]
  "mtime" {with-test & >= "2.0.0"}         [from ocaml/libs/xapi-stdext/lib/xapi-stdext-threads, ocaml/libs/xapi-stdext/lib/xapi-stdext-unix/test]
  "qcheck-core" {with-test & >= "0.21.2"}  [from ocaml/libs/xapi-stdext/lib/xapi-stdext-unix/test]
  "rpclib" {with-test & >= "9.0.0"}        [from ocaml/tests/record_util]
  "rresult" {with-test & >= "0.7.0"}       [from ocaml/libs/xapi-stdext/lib/xapi-stdext-unix/test]
```

Reply:
Unfortunately opam-dune-lint doesn't know how to make changes in .template files, only in dune-project itself.

Reply:
Now the metadata for the package is in dune-project